keras_cv/datasets/pascal_voc/segmentation.py

```diff
@@ -0,0 +1,320 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data loader for Pascal VOC 2012 segmentation dataset.
+
+The image classification and object detection (bounding box) data is covered by existing
+TF datasets in https://www.tensorflow.org/datasets/catalog/voc. The segmentation data (
+both class segmentation and instance segmentation) are included in the VOC 2012, but not
+offered by TF-DS yet. This module is trying to fill this gap while TFDS team can
+address this feature (b/252870855 and https://github.com/tensorflow/datasets/issues/27).
```

Comment:
And also the contribution we have lost with https://github.com/tensorflow/datasets/pull/1198

Reply:
Sure.

Reply:
Added in the latest commit.